src/client/wire_protocol/operations.rs

```diff
@@ -380,10 +403,12 @@ impl Message {
         };
 
         match header.op_code {
-            OpCode::OpReply => Message::read_reply(buffer),
+            OpCode::Reply => {
+                Message::read_reply(buffer, header)
+            },
             opcode => {
-                let s = format!("Expected to read response but instead found: {}", opcode.to_string());
-                Err(s)
+                let s = opcode.to_string();
```

Comment:
Without the temporary variable, the line is nearly 100 characters long, so I wanted to shorten it in some way to make it more readable. Would it be preferable to just use a backslash to split the string onto multiple lines?